rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -558,15 +558,16 @@ def test_punishing_of_oracles_works(inquirer):
                 assert defillama_mock.called is True
 
         # move the current time forward and check that coingecko is still penalized
+        penalty_duration = CachedSettings().get_entry('oracle_penalty_duration')
```

Comment:
you did it here

Reply:
I think the type checker doesn't check tests, so this wasn't flagged

Reply:
Yes, for non typed environments mypy doens't check on the tests